apicast/Roverfile.lock

```diff
@@ -1,8 +1,12 @@
+ansicolors 1.0.2-3
+argparse 0.5.0-1
 busted 2.0.rc12-1
 dkjson 2.5-2
 inspect 3.1.0-1
 ldoc 1.4.6-2
+liquid scm-1
```

Comment:
We will need to work around this before https://github.com/luarocks/luarocks/issues/739 is fixed. Probably by doing the whole clone and copy in rover.

Reply:
Fixed by https://github.com/3scale/lua-rover/pull/2